# Icelandic translation for apport
# Copyright (c) 2008 Rosetta Contributors and Canonical Ltd 2008
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2008.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2012-06-15 07:52+0200\n"
"PO-Revision-Date: 2011-10-07 06:10+0000\n"
"Last-Translator: Stefán Sigurjónsson <stefans@mmedia.is>\n"
"Language-Team: Icelandic <is@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2012-07-07 04:35+0000\n"
"X-Generator: Launchpad (build 15558)\n"
"Language: is\n"

#: ../apport/ui.py:92
msgid "This package does not seem to be installed correctly"
msgstr ""

#. TRANS: %s is the name of the operating system
#: ../apport/ui.py:95
#, python-format
msgid ""
"This is not an official %s package. Please remove any third party package "
"and try again."
msgstr ""

#: ../apport/ui.py:115
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"Þú er með nokkrar úreltar útgáfur af pökkum innsettar. Vinsamlegast uppfærðu "
"eftirfarandi pakka og athugaðu hvort vandamálið endurtekur sig:\n"
"\n"
"%s"

#: ../apport/ui.py:211
msgid "unknown program"
msgstr "óþekkt forrit"

#: ../apport/ui.py:212
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "Fyrirgefðu, forritið \"%s\" hætti óvænt"

#: ../apport/ui.py:213 ../apport/ui.py:1111
#, python-format
msgid "Problem in %s"
msgstr "Vandamál í %s"

#: ../apport/ui.py:214
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"Tölvan þín hefur ekki nóg laust innra minni til að greina sjálfkrafa "
"vandamálið og senda til forritaranna."

#. package does not exist
#: ../apport/ui.py:228 ../apport/ui.py:235 ../apport/ui.py:241
#: ../apport/ui.py:279 ../apport/ui.py:287 ../apport/ui.py:358
#: ../apport/ui.py:361 ../apport/ui.py:559 ../apport/ui.py:926
#: ../apport/ui.py:1065 ../apport/ui.py:1069 ../apport/ui.py:1088
#: ../apport/ui.py:1094
msgid "Invalid problem report"
msgstr "Ógild vandamálsskýrsla"

#: ../apport/ui.py:230 ../apport/ui.py:1071
msgid "This problem report is damaged and cannot be processed."
msgstr "Skýrslan er löskuð og ég get ekki unnið úr henni"

#: ../apport/ui.py:236
msgid "The report belongs to a package that is not installed."
msgstr "Þessi skýrsla tilheyrir pakka sem ekki er uppsettur"

#: ../apport/ui.py:242
msgid "An error occurred while attempting to process this problem report:"
msgstr ""

#: ../apport/ui.py:280
msgid "You are not allowed to access this problem report."
msgstr "Þú hefur ekki leyfi til að skoða þessa skýrslu."

#: ../apport/ui.py:283
msgid "Error"
msgstr "Villa"

#: ../apport/ui.py:284
msgid "There is not enough disk space available to process this report."
msgstr "Það er ekki nóg diskpláss til að vinna úr skýrslunni."

#: ../apport/ui.py:311
msgid "No package specified"
msgstr "Enginn pakki valinn"

#: ../apport/ui.py:312
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""

#: ../apport/ui.py:328
msgid "Invalid PID"
msgstr "Ógilt PID"

#: ../apport/ui.py:329
msgid "The specified process ID does not belong to a program."
msgstr "Þetta ID tilheyrir ekki neinu forriti."

#: ../apport/ui.py:337
msgid "Permission denied"
msgstr "Aðgangur bannaður"

#: ../apport/ui.py:338
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"Tiltekið ferli tilheyrir þér ekki. Vinsamlegast keyrðu þetta forrit sem "
"eigandi ferilsins eða sem kerfisstjórnandi"

#: ../apport/ui.py:359
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr ""

#: ../apport/ui.py:362
#, python-format
msgid "Package %s does not exist"
msgstr "Pakki %s er ekki til"

#: ../apport/ui.py:387 ../apport/ui.py:571 ../apport/ui.py:576
msgid "Cannot create report"
msgstr ""

#: ../apport/ui.py:402 ../apport/ui.py:448 ../apport/ui.py:465
msgid "Updating problem report"
msgstr ""

#: ../apport/ui.py:403
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""

#: ../apport/ui.py:412
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""

#: ../apport/ui.py:449 ../apport/ui.py:466
msgid "No additional information collected."
msgstr ""

#: ../apport/ui.py:516
msgid "What kind of problem do you want to report?"
msgstr "Hvers konar forrit viltu senda skýrslu um?"

#: ../apport/ui.py:533
msgid "Unknown symptom"
msgstr "Óþekkt einkenni"

#: ../apport/ui.py:534
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "Einkennið „%s“ er ekki þekkt."

#: ../apport/ui.py:562
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""

#: ../apport/ui.py:572 ../apport/ui.py:577
msgid "xprop failed to determine process ID of the window"
msgstr ""

#: ../apport/ui.py:591
msgid "%prog <report number>"
msgstr ""

#: ../apport/ui.py:593
msgid "Specify package name."
msgstr ""

#: ../apport/ui.py:595 ../apport/ui.py:644
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr ""

#: ../apport/ui.py:625
msgid ""
"%prog [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""
"%prog [options] [symptom|pid|package|program path|.apport/.crash file]"

#: ../apport/ui.py:628
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""

#: ../apport/ui.py:630
msgid "Click a window as a target for filing a problem report."
msgstr ""

#: ../apport/ui.py:632
msgid "Start in bug updating mode. Can take an optional --package."
msgstr ""

#: ../apport/ui.py:634
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""

#: ../apport/ui.py:636
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""

#: ../apport/ui.py:638
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""

#: ../apport/ui.py:640
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""

#: ../apport/ui.py:642
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""

#: ../apport/ui.py:646
msgid "Print the Apport version number."
msgstr "Gefa Apport útgáfunúmerið."

#: ../apport/ui.py:778
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr ""

#: ../apport/ui.py:779
msgid "Run gdb session"
msgstr ""

#: ../apport/ui.py:780
msgid "Run gdb session without downloading debug symbols"
msgstr ""

#. TRANSLATORS: %s contains the crash report file name
#: ../apport/ui.py:782
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr ""

#: ../apport/ui.py:823
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr ""

#: ../apport/ui.py:927
msgid "Could not determine the package or source package name."
msgstr "Gat ekki ákveðið nafn pakka eða grunnpakka"

#: ../apport/ui.py:945
msgid "Unable to start web browser"
msgstr "Gat ekki ræst vefvafra"

#: ../apport/ui.py:946
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "Gat ekki ræst vefvafra til að opna %s"

#: ../apport/ui.py:1021
#, python-format
msgid "Please enter your account information for the %s bug tracking system"
msgstr "Sláðu inn aðgangsupplýsingar þínar fyrir %s villukerfið."

#: ../apport/ui.py:1032
msgid "Network problem"
msgstr "Vandamál með nettengingu"

#: ../apport/ui.py:1034
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr "Get ekki tengst við gagnagrunn, athugaðu nettenginguna."

#: ../apport/ui.py:1060
msgid "Memory exhaustion"
msgstr "Minnisvandamál"

#: ../apport/ui.py:1061
msgid "Your system does not have enough memory to process this crash report."
msgstr ""
"Kerfið þitt hefur ekki nóg vinnsluminni til að vinna úr þessari skýrslu."

#: ../apport/ui.py:1084 ../apport/ui.py:1093
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr ""

#: ../apport/ui.py:1114
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"Ekki hægt að tilkynna vandamálið:\n"
"\n"
"%s"

#: ../apport/ui.py:1153 ../apport/ui.py:1160
msgid "Problem already known"
msgstr "Vandamál þegar þekkt"

#: ../apport/ui.py:1154
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"Þetta vandamál var þegar tilkynnt í villuskýrslunni sem birtist í "
"vefvafranum. Vinsamlegast athugaðu hvort þú getur bætt við einhverjum "
"upplýsingum sem gætu verið hjálplegar."

#: ../apport/ui.py:1161
msgid "This problem was already reported to developers. Thank you!"
msgstr ""

#: ../bin/apport-retrace.py:37
msgid "%prog [options] <apport problem report | crash ID>"
msgstr "%prog [options] <apport problem report | crash ID>"

#: ../bin/apport-retrace.py:39
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr ""

#: ../bin/apport-retrace.py:41
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""

#: ../bin/apport-retrace.py:43
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr ""

#: ../bin/apport-retrace.py:45
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr ""

#: ../bin/apport-retrace.py:47
msgid "Override report's CoreFile"
msgstr ""

#: ../bin/apport-retrace.py:49
msgid "Override report's ExecutablePath"
msgstr ""

#: ../bin/apport-retrace.py:51
msgid "Override report's ProcMaps"
msgstr ""

#: ../bin/apport-retrace.py:53
msgid "Rebuild report's Package information"
msgstr ""

#: ../bin/apport-retrace.py:55
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""

#: ../bin/apport-retrace.py:57
msgid ""
"Report download/install progress when installing packages into sandbox"
msgstr ""

#: ../bin/apport-retrace.py:59
msgid "Prepend timestamps to log messages, for batch operation"
msgstr ""

#: ../bin/apport-retrace.py:61
msgid "Cache directory for packages downloaded in the sandbox"
msgstr ""

#: ../bin/apport-retrace.py:65 ../debian/tmp/usr/bin/apport-retrace.py:65
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""

#: ../bin/apport-retrace.py:63
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr ""

#: ../bin/apport-retrace.py:65
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""

#: ../bin/apport-retrace.py:67
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""

#: ../bin/apport-retrace.py:69
msgid ""
"Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr ""

#: ../bin/apport-retrace.py:74
msgid "incorrect number of arguments; use --help for a short help"
msgstr ""

#: ../bin/apport-retrace.py:78
msgid ""
"you either need to do a local operation (-s, -g, -o) or supply an "
"authentication file (--auth); see --help for a short help"
msgstr ""

#. translators: don't translate y/n, apport currently only checks for "y"
#: ../bin/apport-retrace.py:110
msgid "OK to send these as attachments? [y/n]"
msgstr "Er allt í lagi að senda þetta sem sem viðhengi? [y/n]"

#: ../gtk/apport-gtk-mime.desktop.in.h:1 ../kde/apport-kde.desktop.in.h:1
#: ../kde/apport-kde-mime.desktop.in.h:1 ../gtk/apport-gtk.desktop.in.h:1
msgid "Report a problem..."
msgstr "Tilkynna vandamál..."

#: ../gtk/apport-gtk-mime.desktop.in.h:2 ../kde/apport-kde.desktop.in.h:2
#: ../kde/apport-kde-mime.desktop.in.h:2 ../gtk/apport-gtk.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "Tilkynna bilun til forritara"

#: ../bin/apport-unpack.py:23
#, python-format
msgid "Usage: %s <report> <target directory>"
msgstr "Notkun: %s <report> <target directory>"

#: ../bin/apport-unpack.py:33
msgid "Destination directory exists and is not empty."
msgstr ""

#: ../data/kernel_oops.py:28
msgid "Your system might become unstable now and might need to be restarted."
msgstr ""

#: ../bin/apport-cli.py:173 ../gtk/apport-gtk.py:154 ../kde/apport-kde.py:163
msgid "Send problem report to the developers?"
msgstr "Senda vandamálsskýrslu til forritara?"

#: ../gtk/apport-gtk.py:163 ../kde/apport-kde.py:171
msgid "Send"
msgstr ""

#: ../gtk/apport-gtk.py:142 ../kde/apport-kde.py:177 ../kde/apport-kde.py:210
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr ""

#: ../gtk/apport-gtk.py:200 ../kde/apport-kde.py:185
#, python-format
msgid "Package: %s"
msgstr ""

#: ../gtk/apport-gtk.py:207 ../kde/apport-kde.py:191
msgid "Sorry, a problem occurred while installing software."
msgstr ""

#: ../gtk/apport-gtk.py:216 ../kde/apport-kde.py:197
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr ""

#: ../gtk/apport-gtk.ui.h:9 ../gtk/apport-gtk.py:221 ../kde/apport-kde.py:203
msgid "Leave Closed"
msgstr ""

#: ../gtk/apport-gtk.py:222 ../kde/apport-kde.py:204 ../kde/apport-kde.py:341
msgid "Relaunch"
msgstr ""

#: ../gtk/apport-gtk.ui.h:10 ../gtk/apport-gtk.py:184 ../gtk/apport-gtk.py:225
#: ../gtk/apport-gtk.py:236 ../kde/apport-kde.py:207 ../kde/apport-kde.py:216
msgid "Continue"
msgstr ""

#: ../gtk/apport-gtk.ui.h:4 ../gtk/apport-gtk.py:233 ../kde/apport-kde.py:213
msgid "If you notice further problems, try restarting the computer."
msgstr ""

#: ../gtk/apport-gtk.py:237 ../kde/apport-kde.py:217
msgid "Ignore future problems of this type"
msgstr ""

#: ../gtk/apport-gtk.py:501 ../kde/apport-kde.py:252
msgid "Hide Details"
msgstr ""

#: ../gtk/apport-gtk.ui.h:7 ../gtk/apport-gtk.py:172 ../gtk/apport-gtk.py:497
#: ../kde/apport-kde.py:255
msgid "Show Details"
msgstr ""

#: ../kde/apport-kde.py:278
msgid "Username:"
msgstr ""

#: ../kde/apport-kde.py:279
msgid "Password:"
msgstr ""

#: ../bin/apport-cli.py:143 ../gtk/apport-gtk.py:118 ../kde/apport-kde.py:322
msgid "(binary data)"
msgstr "(gögn í tvíundakerfi)"

#: ../kde/apport-kde.py:366
msgid "Collecting Problem Information"
msgstr ""

#: ../bin/apport-cli.py:233 ../kde/apport-kde.py:367
msgid "Collecting problem information"
msgstr "Safna saman upplýsingum um vandamál"

#: ../kde/apport-kde.py:368
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""

#: ../kde/apport-kde.py:394
msgid "Uploading Problem Information"
msgstr ""

#: ../bin/apport-cli.py:246 ../gtk/apport-gtk.ui.h:13 ../kde/apport-kde.py:395
msgid "Uploading problem information"
msgstr "Sendi vandamálsskýrslu"

#: ../gtk/apport-gtk.ui.h:15 ../kde/apport-kde.py:396
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"Er að senda upplýsingarnar á villukerfið. Þetta gæti tekið nokkrar mínútur."

#: ../gtk/apport-gtk.ui.h:1 ../kde/apport-kde.py:430 ../kde/apport-kde.py:466
msgid "Apport"
msgstr ""

#: ../bin/apport-cli.py:71
msgid "Press any key to continue..."
msgstr "Ýttu á einhvern lykil til að halda áfram..."

#: ../bin/apport-cli.py:78
msgid "What would you like to do? Your options are:"
msgstr "Hvað viltu gera? Þú getur:"

#: ../bin/apport-cli.py:82
#, python-format
msgid "Please choose (%s):"
msgstr "Vinsamlegast veldu (%s);"

#: ../bin/apport-cli.py:141
#, python-format
msgid "(%i bytes)"
msgstr ""

#: ../bin/apport-cli.py:174
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""

#: ../bin/apport-cli.py:177
#, python-format
msgid "&Send report (%s)"
msgstr "&Senda skýrslu (%s)"

#: ../bin/apport-cli.py:181
msgid "&Examine locally"
msgstr ""

#: ../bin/apport-cli.py:185
msgid "&View report"
msgstr "&Lesa skýrslu"

#: ../bin/apport-cli.py:186
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr "&Geyma skýrslu til að senda inn seinna eða til að færa á annan stað"

#: ../bin/apport-cli.py:187
msgid "Cancel and &ignore future crashes of this program version"
msgstr "Hætta við og h&unsa frekari hrun í þessari útgáfu forritsins"

#: ../bin/apport-cli.py:189 ../bin/apport-cli.py:266 ../bin/apport-cli.py:298
#: ../bin/apport-cli.py:319
msgid "&Cancel"
msgstr "&Hætta við"

#: ../bin/apport-cli.py:217
msgid "Problem report file:"
msgstr "Skrá með upplýsingar um vandamál:"

#: ../bin/apport-cli.py:223 ../bin/apport-cli.py:228
msgid "&Confirm"
msgstr "&Staðfesta"

#: ../bin/apport-cli.py:227
#, python-format
msgid "Error: %s"
msgstr "Villa: %s"

#: ../bin/apport-cli.py:234
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"Þú getur sent inn skýrslu með þessum upplýsingum til að bæta forritið.\n"
"Þetta getur tekið nokkra stund."

#: ../bin/apport-cli.py:247
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""

#: ../bin/apport-cli.py:297
msgid "&Done"
msgstr "&Lokið"

#: ../bin/apport-cli.py:303
msgid "none"
msgstr "ekkert"

#: ../bin/apport-cli.py:304
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "Valið: %s. Margir valkostir:"

#: ../bin/apport-cli.py:320
msgid "Choices:"
msgstr "Valkostir:"

#: ../bin/apport-cli.py:334
msgid "Path to file (Enter to cancel):"
msgstr "Slóð skráar (Enter til að hætta við):"

#: ../bin/apport-cli.py:340
msgid "File does not exist."
msgstr "Skrá er ekki til."

#: ../bin/apport-cli.py:342
msgid "This is a directory."
msgstr "Þetta er mappa."

#: ../bin/apport-cli.py:348
msgid "To continue, you must visit the following URL:"
msgstr "Þú verður að heimsækja eftirfarandi slóð til að halda áfram:"

#: ../bin/apport-cli.py:350
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"Þú getur ræst vafra núna eða afritað þessa slóð í vafra á annari tölvu."

#: ../bin/apport-cli.py:352
msgid "Launch a browser now"
msgstr "Ræsa netvafra"

#: ../bin/apport-cli.py:366
msgid "No pending crash reports. Try --help for more information."
msgstr ""

#: ../data/apportcheckresume.py:68
msgid ""
"This occured during a previous suspend and prevented it from resuming "
"properly."
msgstr ""

#: ../data/apportcheckresume.py:70
msgid ""
"This occured during a previous hibernate and prevented it from resuming "
"properly."
msgstr ""

#: ../data/apportcheckresume.py:75
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""

#: ../gtk/apport-gtk.ui.h:2
msgid "Crash report"
msgstr ""

#: ../gtk/apport-gtk.ui.h:3
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr ""

#: ../gtk/apport-gtk.ui.h:5
msgid "Send an error report to help fix this problem"
msgstr ""

#: ../gtk/apport-gtk.ui.h:6
msgid "Ignore future problems of this program version"
msgstr ""

#: ../gtk/apport-gtk.ui.h:8
msgid "_Examine locally"
msgstr ""

#: ../gtk/apport-gtk.ui.h:11
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>Safna upplýsingum um vandamál</b></big>"

#: ../gtk/apport-gtk.ui.h:12
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"Er að safna upplýsingum sem gætu hjálpað forriturum að laga vandann sem þú "
"tilkynnir"

#: ../gtk/apport-gtk.ui.h:14
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>Sendi vandamálsskýrslu</b></big>"

#: ../gtk/apport-gtk.py:133
#, python-format
msgid "Sorry, the application %s has closed unexpectedly."
msgstr ""

#: ../gtk/apport-gtk.py:136
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr ""

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Apport villuskrá"

#, no-c-format, python-format
#~ msgid "Complete report (recommended; %s)"
#~ msgstr "Full skýrsla (mælt með; %s)"

#~ msgid ""
#~ "<big><b>Send problem report to the developers?</b></big>\n"
#~ "\n"
#~ "After the problem report has been sent, please fill out the form in the "
#~ "automatically opened web browser."
#~ msgstr ""
#~ "<big><b>Senda vandamálsskýrslu til forritaranna?</b></big>\n"
#~ "\n"
#~ "Eftir að skýrslan hefur verið send, vinsamlegast fylltu út formið í vafranum "
#~ "sem opnast sjálfkrafa."

#~ msgid ""
#~ "If you were not doing anything confidential (entering passwords or other "
#~ "private information), you can help to improve the application by reporting "
#~ "the problem."
#~ msgstr ""
#~ "Ef þú varst ekki að gera eitthvað leynilegt (slá inn lykilorð eða aðrar "
#~ "persónulegar upplýsingar) geturðu hjálpað við að bæta forritið með því að "
#~ "tilkynna vandamálið."

#~ msgid "Application problem"
#~ msgstr "Forritsvandamál"

#~ msgid "_Send Report"
#~ msgstr "_Senda skýrslu"

#~ msgid ""
#~ "If you were not doing anything confidential (entering passwords or other\n"
#~ "private information), you can help to improve the application by reporting\n"
#~ "the problem."
#~ msgstr ""
#~ "Ef þú varst ekki að gera eitthvað persónugreinanlegt (slá inn lykilorð eða \n"
#~ "aðrar persónulegar upplýsingar) getur þú hjálpað við að laga forritið með\n"
#~ "því að tilkynna vandamálið."

#~ msgid "Restart _Program"
#~ msgstr "_Endurræsa Forrit"

#, no-c-format
#~ msgid ""
#~ "<span weight=\"bold\" size=\"larger\">Sorry, the package \"%s\" failed to "
#~ "install or upgrade.</span>"
#~ msgstr ""
#~ "<span weight=\"bold\" size=\"larger\">Því miður tókst ekki að setja upp eða "
#~ "uppfæra pakkann \"%s\"</span>"

#, no-c-format, python-format
#~ msgid "Reduced report (slow Internet connection; %s)"
#~ msgstr "Minnkuð skýrsla (hæg nettenging; %s)"

#~ msgid "_Ignore future crashes of this program version"
#~ msgstr "_Hunsa frekari hrun þessarar forritsútgáfu"

#~ msgid ""
#~ "This will remove some large items from the report. These are very useful for "
#~ "developers to debug the problem, but might be too big for you to upload if "
#~ "you have a slow internet connection."
#~ msgstr ""
#~ "Þetta tekur nokkra stóra hluta úr skýrslunni. Þeir eru gagnlegir forriturum "
#~ "til að greina vandamálkið, en gætu verið of stórir til að að senda yfir hæga "
#~ "nettengingu."

#~ msgid "&Report Problem..."
#~ msgstr "&Tilkynna vandamál"

#~ msgid "_Report Problem..."
#~ msgstr "_Tilkynna vandamál..."

#~ msgid "Kernel problem"
#~ msgstr "Kjarnavandi"

#~ msgid ""
#~ "You can help the developers to fix the package by reporting the problem."
#~ msgstr ""
#~ "Þú getur hjálpað forriturunum að laga pakkann með því að senda inn "
#~ "tilkynningu."

#, python-format
#~ msgid "This is not a genuine %s package"
#~ msgstr "Þetta er ekki alvöru %s pakki"

#~ msgid "Content of the report"
#~ msgstr "Innihald skýrslunnar"

#~ msgid "Package problem"
#~ msgstr "Pakkavandamál"

#, python-format
#~ msgid "The package \"%s\" failed to install or upgrade."
#~ msgstr "Það tókst hvorki að setja upp “%s“ eða uppfæra."

#, python-format
#~ msgid "%s closed unexpectedly on %s at %s."
#~ msgstr "%s lokaði óviðbúið þann %s klukkan %s."

#~ msgid "Your system encountered a serious kernel problem."
#~ msgstr "Kerfið þitt lenti í alvarlegum vanda með stýrikerfiskjarnann"

#~ msgid "You can help the developers to fix the problem by reporting it."
#~ msgstr "Þú getur hjálpað til með því að tilkynna vandann."

#, python-format
#~ msgid "&Send complete report (recommended; %s)"
#~ msgstr "&Senda alla skýrsluna (mælt með þessu; %s)"

#, python-format
#~ msgid "Send &reduced report (slow Internet connection; %s)"
#~ msgstr "Senda &stytta skýrslu (hæg nettenging; %s)"
